darwin/dataset/remote_dataset_v2.py

```diff
@@ -832,3 +856,18 @@ def register_multi_slotted(
                 print(f"  - {item}")
         print(f"Reistration complete. Check your items in the dataset: {self.slug}")
         return results
+
+    @retry(
+        wait=wait_exponential_jitter(initial=30, max=240),
+        stop=stop_after_attempt(5),
+        retry=retry_if_exception_type(requests.exceptions.RequestException),
+        before_sleep=before_sleep,
```

Comment:
This is just a 1st draft policy, I'll need feedback on the optimal min/max times as well as the criteria we stop after